src/pages/en/migrate.md

```diff
@@ -63,19 +63,21 @@ Read [RFC0017](https://github.com/withastro/rfcs/blob/main/proposals/0017-markdo
 
 This includes a few changes to be aware of:
 
-- **BREAKING:** `<script hoist>` is the new default `<script>` behavior. The `hoist` attribute has been removed.
+- **BREAKING:** `<script hoist type="module">` is the new default `<script>` behavior. The `hoist` and `type="module"` attributes have been removed.
```

Comment:
```suggestion
- **BREAKING:** `<script hoist>` is the new default `<script>` behavior. The `hoist` attribute has been removed. To use the new default behaviour, make sure there are no other attributes on the `<script>` tag. For example remove `type="module"` if you were using it before.
```

Reply:
I prefer this one over the suggestion from Fuzzy because of how the default is removed from the script.

From the WIP version of `debugging.md`:
> If you have included the [`is:inline` directive](https://docs.astro.build/en/reference/directives-reference/#isinline) or any other attribute in your tag, this default behavior is removed.